doc/adr/0004-no-repo-creation.md

```diff
@@ -0,0 +1,19 @@
+# 4. No repo creation
+
+Date: 2021-07-06
+
+## Status
+
+Proposed
+
+## Context
+
+In the initial pre-alpha version of weave-gitops and WKP, we created git repositories for the end-user.  However, we encounter more instances where customers don't permit developers, application or platform SREs to create repositories in the customer's organization.  While this is a convenient feature, we've decided the effort and support required are better focused elsewhere in the application.
```

Comment:
Nit: I don't think the second sentence scans very well. Maybe: "However, we have encountered more (or many)" or just "However, we have encountered instances..."